### PR TITLE
fix: remove <base />

### LINKED ISSUE #649: [BUG]: wrong anchor link in the website
The website's anchor links appear to be incorrect, missing the original path, such as /blog/introducing-pages-router, as shown in the picture.

CleanShot 2024-04-04 at 23 03 40@2x

### PR DESCRIPTION
**Pull request by Aslemammad**
Resolves #649

### CODE REVIEW

File: packages/waku/src/lib/plugins/vite-plugin-rsc-index.ts [hunk 1]
```diff
@@ -75,9 +75,6 @@ ${opts.htmlHead}
     },
     transformIndexHtml() {
       return [
-        // HACK without <base>, some relative assets don't work.
```
Comment (dai-shi) [inline packages/waku/src/lib/plugins/vite-plugin-rsc-index.ts:78]: Do you know what was the original problem?

Comment (Aslemammad) [reply packages/waku/src/lib/plugins/vite-plugin-rsc-index.ts:78]: No honestly, but what I guess was the potential solution was the fact the we made things either absolute to the project root or to the user's root.

Comment (dai-shi) [reply packages/waku/src/lib/plugins/vite-plugin-rsc-index.ts:78]: Let me check:
- It's introduced in #289.
- https://github.com/dai-shi/waku/pull/289/commits/cb17af57774fc033d6eb8a26d410a62bad11d446
- Going back to 7c0a6ef4fc8ff3aa2d56d0728882655534d14b0d and remove the hack
- npm run website:prd
- open http://localhost:8080/blog/introducing-waku

image

Now, let's try this branch:
- git switch fix/remove-base 
- npm run website:prd
- open http://localhost:8080/blog/introducing-waku

same error

image

So, no, it's not fixed.

Comment (Aslemammad) [reply packages/waku/src/lib/plugins/vite-plugin-rsc-index.ts:78]: will check this soon!

Comment (Aslemammad) [reply packages/waku/src/lib/plugins/vite-plugin-rsc-index.ts:78]: @PerfectPan For sure! feel free to, i'd be happy to close this if that solves it.

**Comment by dai-shi** [comment]
Superseded by #697.

**Pull request closed without merging**